libfastpm/thermalvelocity.c

```diff
@@ -375,12 +375,26 @@ fastpm_split_ncdm(FastPMncdmInitData * nid,
             //give id, mass and add thm vel
             dest->id[r] = j * src->meta._q_size + src->id[i];
             dest->mass[r] = nid->mass[j] / (nid->m_ncdm_sum / nid->n_ncdm) * M0;
-
+            
             for(d = 0; d < 3; d ++){
-                /* conjugate momentum unit [a^2 xdot, where x is comoving dist] */
+                // conjugate momentum unit [a^2 xdot, where x is comoving dist]
```

Comment:
Use C style comments as much as possible to be consistent with old code.

Reply:
I didn't realise this was a thing, but just read up on it. So should I avoid // in general?

Reply:
Consistency is a thing -- if the file already uses `/**/` everywhere, then a sudden `//` looks out of place.

I typically used `//` only for commenting out diagnostic source lines, and `/**/` for real comments.